src/components/data-controllers/SingleDataController/SingleDataController.js

```diff
@@ -177,6 +177,37 @@ export default createReactClass({
       this.state.dataTableTimeOfYear
     );
 
+    // Spec for generating tabs containing various graphs.
+    // Property names indicate whether the dataset is a multi-year mean or not.
+    const graphTabsSpec = {
```

Comment:
This is pretty neat! Is the plan to do it in all the datacontrollers eventually, given time enough?

Reply:
Heh, no time like the present.